src/utilities/api/types.ts

```diff
@@ -6,12 +6,25 @@ export type PaymentParams = {
   seller_id: string;
 };
 
-export type Merchant = {
-  name: string;
-  email: string;
-  address: string;
+export type Address = {
+  address1: string;
+  address2: string;
   city: string;
-  stateForm: string;
-  zipCode: string;
-  showPayModal: boolean;
+  phone_number: string;
+  state: string;
+  zip_code: number;
+};
+
+export type Seller = {
+  addresses?: Address[];
+  //TO DO: what is className doing
+  className: '';
+  cuisineName: string;
+  email?: string;
+  name: string;
+  
+  //TO DO: why is this necessary here
+  showPayModal?: boolean;
+  summary: string;
+  story: string;
 };
```

Comment:
Yeah it looks like we need ownerName, acceptsDonations, sellsGiftCards, ownerImageUrl